src/Moryx.Runtime.Maintenance.Web/packages.config

```diff
@@ -1,9 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Moryx" version="3.0.0-dev.42" targetFramework="net461" />
-  <package id="Moryx.Runtime" version="3.0.0-dev.42" targetFramework="net461" />
-  <package id="Moryx.Runtime.Maintenance" version="3.0.0-dev.42" targetFramework="net461" />
-  <package id="Moryx.Tools.Wcf" version="3.0.0-dev.42" targetFramework="net461" />
+  <package id="EntityFramework" version="6.4.4" targetFramework="net461" />
```

Comment:
Here only EntityFramework is referenced

Reply:
The SqlServer reference is part of the EntityFramework package